cores/cv32e40p/user_manual/source/fpu.rst

```diff
@@ -99,27 +32,8 @@ exceptions that occurred since it was last reset and the rounding mode.
 :ref:`csr-fflags` and :ref:`csr-frm` can be accessed directly or via :ref:`csr-fcsr` which is mapped to
 those two registers.
 
-Since CV32E40P includes an iterative div/sqrt unit, its precision and
-latency can be controlled via the custom :ref:`csr-fprec` CSR. This allows faster
-division / square-root operations at the lower precision. By default,
-the single-precision equivalents are computed with a latency of 8
-cycles. The FPU CSRs are further described in :ref:`cs-registers`.
-
 Floating-point Performance Counters:
```

Comment:
What is this section about; should it be removed all together? If not, which counters/CSRs are being talked about here?

Reply:
This says what was the original intention of the FPREC CSR. a csr that has been removed anyway. So it's a deprecated section

Reply:
Ah it's referred to the following section. Yes removing it